rs/ledger_suite/icp/archive/src/main.rs

```diff
@@ -251,14 +251,14 @@ fn encode_metrics(w: &mut MetricsEncoder<Vec<u8>>) -> std::io::Result<()> {
         "Size of the stable memory allocated by this canister measured in 64K Wasm pages.",
     )?;
     w.encode_gauge(
-        "archive_node_stable_memory_bytes",
+        "stable_memory_bytes",
         (stable_memory_size_in_pages() * 64 * 1024) as f64,
         "Size of the stable memory allocated by this canister measured in bytes.",
     )?;
     w.encode_gauge(
-        "archive_node_total_memory_bytes",
+        "heap_memory_bytes",
         total_memory_size_bytes() as f64,
```

Comment:
nit: I would rename this method `total_memory_size_bytes()` for something like `heap_memory_bytes()` since I find it very confusing (it's not the total memory but only the heap) (also in the other files below if it's not the same method)

Reply:
Renamed, thanks for the suggestion!